comms/src/peer_manager/peer.rs

```diff
@@ -71,6 +71,7 @@ pub struct Peer {
     /// Flags for the peer.
     pub flags: PeerFlags,
     pub banned_until: Option<NaiveDateTime>,
+    pub banned_reason: String,
```

Comment:
This is a breaking change - we can write migrations for this (the infrastructure is there - see [here](https://github.com/tari-project/tari/blob/690b9ee09750c8e8f4d44c1a916a476c082a727f/comms/src/peer_manager/migrations/v1.rs#L68)) so users (esp mobile) are not disrupted, or we need to require that peer DBs are deleted.